containers/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#set -o xtrace
+
+# Necessary to open windows
+xhost local:root
+
+# Absolute path to this script
+SCRIPT=$(readlink -f $0)
+# Absolute path this script is in
+SCRIPTPATH=`dirname $SCRIPT`
+
+# Get the PlotJuggler repo path to mount in the container
+MOUNT_LOCAL_REPO="-v $SCRIPTPATH/..:/root/airunner"
+# Optional nvidia runtime, needs nvidia runtime installed (https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+#RUNTIME=--runtime=nvidia
+RUNTIME=
+# Note: privileged, net host, and devices may be an overkill
+
+docker run \
```

Comment:
i will add all my windows stuff and also take a look at implementing some of the things you're showing here. i've had success running everything on docker in ubuntu and also wine but right now my wine build is broken. work towards getting that into the main repo asap